### PR TITLE
new todo-backend

### PR DESCRIPTION
**Pull request by gjbianco**

### CODE REVIEW

File: todo-backend/src/entities/db.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const {
+  DATABASE_NAME,
+  DATABASE_USER,
+  DATABASE_PASSWORD,
+  DATABASE_SVC,
+  DATABASE_PORT,
+} = process.env;
+
+export const dbConnectionOptions: Options = {
```
Comment (rsriniva) [inline todo-backend/src/entities/db.ts:11]: these consts are potential candidates for configmaps and secrets via dotenv pkg?

File: todo-backend/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import app from "@server";
+
+// Start the server
+const port = Number(process.env.PORT || 8080);
```
Comment (rsriniva) [inline todo-backend/src/index.ts:4]: potential configmap variable?

**Review by rsriniva — COMMENTED** [review]
nice and sweet. looks good

File: todo-backend/src/entities/db.ts (hunk 1 above)
Comment (gjbianco) [reply todo-backend/src/entities/db.ts:11]: Yep, these should pick up the values from their corresponding environment variables.

File: todo-backend/src/index.ts (hunk 2 above)
Comment (gjbianco) [reply todo-backend/src/index.ts:4]: Yep, like above, as long as the environment variable gets set in the pod, the application should pick up the variable.

**Review by rsriniva — APPROVED** [review]
Approved. Merge. I will QA when complete lab is ready

**Pull request merged**